fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/highlight/FDBLuceneHighlightingTest.java

```diff
@@ -101,6 +101,22 @@ void highlightedBitsetQuery() {
         }
     }
 
+    @Test
```

Comment:
Other types that are included in the change are the rest of the subclasses of PointRange (BigInteger, bBinary, Double etc). They should be tested.